queries.sql

```diff
@@ -0,0 +1,110 @@
+PRAGMA foreign_keys = ON;
+
+DROP TABLE IF EXISTS channel_user;
+DROP TABLE IF EXISTS message;
+DROP TABLE IF EXISTS channel;
+DROP TABLE IF EXISTS organization;
+DROP TABLE IF EXISTS user;
+
+CREATE TABLE organization (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    name VARCHAR(128) NOT NULL
+);
+
+CREATE TABLE channel (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    name VARCHAR(128) NOT NULL,
+    organization_id INTEGER REFERENCES organization(id)
+);
+
+CREATE TABLE user (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    name VARCHAR(128) NOT NULL
+);
+
+CREATE TABLE message (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    post_time DATETIME default current_time,
+    content TEXT,
+    user_id INTEGER REFERENCES user(id),
+    channel_id INTEGER REFERENCES channel(id)
+);
+
+CREATE TABLE channel_user (
+    channel_id INTEGER REFERENCES channel(id),
+    user_id INTEGER REFERENCES user(id)
+);
+
+INSERT INTO organization (name) VALUES ('Lambda School');
+
+INSERT INTO user (name) VALUES ('Alice');
+INSERT INTO user (name) VALUES ('Bob');
```

Comment:
Can also use insert multiple values in one line `INSERT INTO user (name) VALUES ('Alice'), ('Bob'), ('Chris')`